main.go

```diff
@@ -61,4 +66,6 @@ func init() {
 		"js", false, "Enables EnableJavascript execution await")
 	rootCmd.PersistentFlags().IntVar(&scraperParameters.Timeout,
 		"timeout", 0, "If > 0, specify a timeout (seconds) for js execution await")
+	rootCmd.PersistentFlags().BoolVarP(&csv,
+		"output-csv", "c", false, "Output csv: <url>,<email>")
```

Comment:
If going forward with multiple formats, I think we should go with `output-format` flag and have options as "stdout (default)", "csv" and later "json", "xlsx" and etc.

@f00b455 🤔